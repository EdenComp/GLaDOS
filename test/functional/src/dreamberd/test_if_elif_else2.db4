str bl = "
";

fn printLn(s) {
    print(s);
    print(bl);
}

fn intCmp(a, b) {
    if (a == b) {
        return "a == b";
    } elif (a < b) {
        return "a < b";
    } else {
        return "a > b";
    }
}

fn main() {
    int a = 1;
    int b = 2;
    int c = 3;

    printLn(a);
    printLn(b);
    printLn(c);
    str s = intCmp(a, b);
    printLn(s);
    str ss = intCmp(c, a);
    printLn(ss);
    str sss = intCmp(a, a);
    printLn(sss);
}

main();