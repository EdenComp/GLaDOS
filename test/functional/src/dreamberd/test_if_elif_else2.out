1
2
3
a < b
a > b
a == b

